setup_py_upgrade.py

```diff
@@ -134,11 +134,23 @@ def _list_as_str(lst: Sequence[str]) -> str:
         return '\n' + '\n'.join(lst)
 
 
-def reformat_lists(section: Dict[str, Any]) -> Dict[str, Any]:
-    return {
-        k: _list_as_str(v) if isinstance(v, (list, tuple)) else v
-        for k, v in section.items()
-    }
+def _dict_as_str(dct: Dict[str, str]) -> str:
+    return '\n' + '\n'.join(
+        (key + ' = ' + value) for key, value in dct.items()
+    )
+
+
+def reformat_values(section: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
maybe name this `_reformat` -- now that it's doing a bunch of things the original name made no sense and I don't think `_values` adds any value ;)